library/platform.c

```diff
@@ -70,28 +70,15 @@ int mbedtls_platform_set_calloc_free( void * (*calloc_func)( size_t, size_t ),
 }
 #endif /* MBEDTLS_PLATFORM_MEMORY */
 
-#if defined(_WIN32)
+#if defined(MSVC_LESS_THAN_14)
```

Comment:
I don't think this will work on MinGW. MinGW will define `_WIN32`but not MSVC.

However - I also don't think you need to know. If you keep `#if defined(_WIN32)`, there will still be a call to `mbedtls_vsnprintf()` and what happens there you can work out locally.